.github/workflows/book.yml

```
@@ -15,16 +15,27 @@ jobs:
           toolchain: nightly
           override: true
 
-      - name: Setup mdBook
-        uses: peaceiris/actions-mdbook@v1
-        with:
-          mdbook-version: 'latest'
+      # - name: Setup mdBook
+      #   uses: peaceiris/actions-mdbook@v1
+      #   with:
+      #     mdbook-version: 'latest'
+
+      # mdBook for mdbook-pdf compatibility by fixing broken links in print.html
+      # Waiting for https://github.com/rust-lang/mdBook/pull/1738 to get merged
+      - name: Clone patched mdbook source code
+        run: cd .. && git clone https://github.com/HollowMan6/mdBook
+
+      - name: Install patched mdbook
+        run: cd ../mdBook && cargo install --path . --force
```

And in fact the manual clone is unnecessary. Instead, you can install directly from a git path:
```suggestion
        run: cargo install --git https://github.com/HollowMan6/mdBook.git --rev 62e01b34c23b957579c04ee1b24b57814ed8a4d5
```

---

OKay!